engine/src/test/java/org/camunda/bpm/engine/test/bpmn/scripttask/ScriptTaskTest.java

```diff
@@ -402,12 +407,23 @@ public void testGroovyVariable() {
 
   @Test
   public void testJuelExpression() {
-    deployProcess(JUEL, "${execution.setVariable('foo', 'bar')}");
+    deployProcess(Bpmn.createExecutableProcess("testProcess")
+      .startEvent()
+        .exclusiveGateway()
+          .condition("true", "${list.contains('foo')}")
+          .userTask("userTask")
+        .moveToLastGateway()
+          .condition("false", "${!list.contains('foo')}")
+          .endEvent()
+    .done());
 
-    ProcessInstance pi = runtimeService.startProcessInstanceByKey("testProcess");
+    ProcessInstance pi = runtimeService.startProcessInstanceByKey("testProcess",
+        Variables.createVariables().putValue("list", Arrays.asList("foo")));
 
-    String variableValue = (String) runtimeService.getVariable(pi.getId(), "foo");
-    assertEquals("bar", variableValue);
+    HistoricActivityInstance userTask = historyService.createHistoricActivityInstanceQuery()
+        .activityId("userTask")
+        .singleResult();
+    assertThat(userTask).isNotNull();
```

Comment:
Can we revert these changes? We shouldn't replace `ScriptTaskTest#testJuelExpression` and we cover this scenario already in `ExpressionManagerTest#testJuelExpressionWithNonPublicClass`.